subsys/testsuite/Kconfig

```diff
@@ -13,6 +13,9 @@ config TEST
 	help
 	  Mark a project or an application as a test. This will enable a few
 	  test defaults.
+	# For tests, store thread names in binary and dump them on crash to
```

Comment:
can you please put the comment the select on top of the help, this looks confusing and the comment seem to be part of the help.

Reply:
> can you please put the comment the select on top of the help

Done.